contributing.rst

```diff
@@ -311,16 +314,7 @@ When you clone down a repo, the local copy calls your GitHub copy ``origin``. Yo
 
 .. code-block:: console
 
-  $ git remote add --track upstream https://github.com/opendatakit/docs.git
-  
-.. note::
-
-  If you are working on windows use:
-  
-  .. code-block:: console
-  
-    >git remote add upstream https://github.com/opendatakit/docs.git
-  
+  $ git remote add --track upstream https://github.com/opendatakit/docs.git  
```

Comment:
This command  does not work on windows, However, git branch --set-upstream master origin/master can be used to keep the track .